### ISSUE TITLE
As a browser, I'd like to understand if a grant is likely to have ended

### ISSUE THREAD

**Issue by stevieflow** [OpenDataServices/grantnav#221]
Given that GrantNav has data from over 10 years, I'd like to understand if some records are likely to represent grants that have since lapsed / are over

I'd appreciate if GrantNav could tell me this, based on a calculation between the start/award date and duration (months).  If it seems that the grant is "old", I'd like GrantNav to tell me something in plain English:

> This grant was awarded X years/months ago.  It was due to last for (months).  It's therefore likely that this grant has been completed.

Where grants do not include the duration (months) data, I'd like GrantNav to tell me how long ago it was awarded - and provide some indication as to how "fresh" it might be:

> This grant was awarded X years/months ago.  (If over x months) It could be that this grant has now ended.

**Comment by morchickit** [comment]
This is connected to #28

**Comment by morchickit** [comment]
This will be good to look at the standard and add status field. https://github.com/ThreeSixtyGiving/standard/issues @stevieflow  to add to the standard repo

**Comment by stevieflow** [comment]
@morchickit actually - it's been a long standing issue in the standard: https://github.com/ThreeSixtyGiving/standard/issues/10

**Comment by morchickit** [comment]
Ok, great. At least now we know the connection :-)

**Comment by morchickit** [comment]
Looking it now again, I think the GrantNav can't really assume that a Grant is over by simple date calculation since some grants can be extended. Tagging @KDuerden here for her input.

**Comment by robredpath** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Determining grant status](https://trello.com/c/wqYQXpAu/28-determining-grant-status)
